core/ordering/cosipbft/blocksync/default.go

```diff
@@ -17,10 +17,17 @@ import (
 	"go.dedis.ch/dela/core/ordering/cosipbft/pbft"
 	otypes "go.dedis.ch/dela/core/ordering/cosipbft/types"
 	"go.dedis.ch/dela/crypto"
+	"go.dedis.ch/dela/internal/tracing"
 	"go.dedis.ch/dela/mino"
 	"golang.org/x/xerrors"
 )
 
+var (
+	// ProtocolName denotes the value of the protocol span tag associated with
+	// the `blocksync` protocol.
+	ProtocolName = "blocksync"
```

Comment:
Do you really need to have it publicly declared ? The only place is in cosipbft/mod.go, which seems superfluous.

Reply:
Made it public because it was used in s.sync.Sync which now seems redundant. Gonna make these and the ones below private